modules/gltf/editor/editor_import_blend_runner.h

```diff
@@ -60,6 +61,7 @@ class EditorImportBlendRunner : public Node {
 	bool is_running() { return blender_pid != 0 && OS::get_singleton()->is_process_running(blender_pid); }
 	bool is_using_rpc() { return rpc_port != 0; }
 	Error do_import(const Dictionary &p_options);
+	HTTPClient::Status connect_blender_rpc(Ref<HTTPClient>, const int timeout_usecs);
```

Comment:
```suggestion
	HTTPClient::Status connect_blender_rpc(Ref<HTTPClient> p_client, int p_timeout_usecs);
```
Match arguments, and use `p_`, also no `const` for simple types